### DISCUSSION TITLE
this tool doesnt clear the extra space in react's className

### DISCUSSION THREAD

**Discussion by IPreferToEatDinner** [tailwindlabs/tailwindcss#12457] [Help]
I used prettier-plugin-tailwindcss, prettier and postcss correctly, and I did implement class name typography when formatting, but I didn't implement the first and last spaces and middle spaces of the className

just like this:
```HTML
<div className="flex  justify-start   h-10   "></div>
```
to

```HTML
 <div className="flex  h-10   justify-start   "></div>
```

how to solve this problem

**Comment by makibat** [comment]
[Removal of extra whitespace is not supported atm](https://github.com/tailwindlabs/tailwindcss/discussions/7560#discussioncomment-4668210)
